geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/UpdateGeoMesaRecord.scala

```diff
@@ -0,0 +1,15 @@
+/***********************************************************************
+ * Copyright (c) 2015-2021 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.geomesa.nifi.datastore.processor
+
+import org.apache.nifi.annotation.documentation.{CapabilityDescription, Tags}
+
+@Tags(Array("geomesa", "geo", "update", "records", "geotools"))
+@CapabilityDescription("Update existing features in GeoMesa")
+class UpdateGeoMesaRecord extends RecordUpdateProcessor
```

Comment:
Ok, so these "little" classes are needed since RecordUpdateProcessor can still be mixed in with another trait/interface?

(And once the deprecated classes are removed, then one can unwind this and clean it up?)